config/sources/families/rockchip64.conf

```diff
@@ -7,6 +7,14 @@ case $BRANCH in
 	legacy)
 		KERNELSOURCE='https://github.com/ayufan-rock64/linux-kernel'
 		KERNELBRANCH='tag:4.4.202-1237-rockchip-ayufan'
+
+		if [[ $BOARD == station-p1 ]]; then
```

Comment:
It still needs to be a single config for `rockchip64` family.
Changes needed for station-m1/p1 should be incorporated into `linux-rockchip64-legacy.config`